### PR TITLE
Update es-ES.tModLoader.json

### PR DESCRIPTION
**Pull request by Igmc**
Tweaked some strings of the Spanish localization

### CODE REVIEW

File: patches/tModLoader/Terraria.Localization.Content/es-ES.tModLoader.json [hunk 1]
```diff
@@ -154,7 +154,7 @@
 		"MBUpdateAll": "Actualizar todos",
 		"MBReloadBrowser": "Recargar Navegador",
 		"MBGettingData": "Obteniendo datos...",
-		"MBPopulatingBrowser": "Rellenando navegador...",
+		"MBPopulatingBrowser": "Completando navegador...",
```
Comment (Jofairden) [inline patches/tModLoader/Terraria.Localization.Content/es-ES.tModLoader.json:157]: Are you sure 'Completing browser' is the right translation? That's a bit of an odd one..

Comment (Igmc) [reply patches/tModLoader/Terraria.Localization.Content/es-ES.tModLoader.json:157]: Mmm maybe that's right. Yeah, leave Rellenando. 
I thought that the second one was a lil better yesterday, as it was a synonim, but now thinking on what does it refers to, uh, I'm not sure.
Sorry about that.

Comment (Jofairden) [reply patches/tModLoader/Terraria.Localization.Content/es-ES.tModLoader.json:157]: If you revert it, I can accept the PR. Rest looks fine

Comment (Igmc) [reply patches/tModLoader/Terraria.Localization.Content/es-ES.tModLoader.json]: Okey, done.

**Pull request merged**